dev-lang/luajit/luajit-2.0.5-r2.ebuild

```diff
@@ -38,10 +38,16 @@ _emake() {
 		PREFIX="${EPREFIX}/usr" \
 		MULTILIB="$(get_libdir)" \
 		DESTDIR="${D}" \
+		CFLAGS="" \
+		LDFLAGS="" \
 		HOST_CC="$(tc-getBUILD_CC)" \
+		HOST_CFLAGS="${BUILD_CPPFLAGS} ${BUILD_CFLAGS}" \
```

Comment:
I don't think that adding CPPFLAGS here is correct

Reply:
Using CPPFLAGS was suggested by @vapier otherwise we can't add custom CPPFLAGS to the build if needed.